### PR TITLE
Bump `nn.functional.conv3d` tolerances for `test_comprehensive`

### PR DESCRIPTION
**Pull request by eqy**
`float16` tolerance was previously set to `1e-5` which seemed very low

### CODE REVIEW

**Comment by albanD** [comment]
You mean that our default fp16 tolerance is `1e5`? That does sound suprising indeed?

**Comment by eqy** [comment]
Yes, as that seems to be the `atol` default here: https://github.com/pytorch/pytorch/blob/5d964a5eb71614d9ddcea65e3417ac25ca69dada/torch/testing/_comparison.py#L1364, though `rtol=1e-3` would probably cover many cases
There are custom tolerances for this test set, but only for `float32`: https://github.com/pytorch/pytorch/blob/118d7e14806d08894dd0e0609ad585f3d8a82e5e/test/inductor/test_torchinductor_opinfo.py#L714

**Review by albanD — COMMENTED** [review]
Ok

**Comment by nWEIdia** [comment]
Please fix lint :) and then let's try merge it.

**Review by Chillee — APPROVED** [review]

**Review by albanD — APPROVED** [review]
Ok!
